fe/fe-core/src/main/java/com/starrocks/persist/StorageInfo.java

```diff
@@ -34,11 +34,14 @@
 
 package com.starrocks.persist;
 
+import com.google.gson.annotations.SerializedName;
+
 /**
  * This class is designed for sending storage information from master to standby master.
  * StorageInfo is easier to serialize to a Json String than class Storage
  */
 public class StorageInfo {
+    @SerializedName("imageJournalId")
```

Comment:
does this mean, the change should apply to 3.2.x in order to make it work to upgrade to v3.3?

Reply:
Gson can ignore field in class in default